client/landing/stepper/declarative-flow/site-migration-flow.ts

```diff
@@ -32,8 +32,9 @@ const siteMigration: Flow = {
 	isSignupFlow: false,
 
 	useSideEffect() {
-		const { setIntent } = useDispatch( ONBOARD_STORE );
+		const { setIntent, resetOnboardStore } = useDispatch( ONBOARD_STORE );
 		useEffect( () => {
+			resetOnboardStore();
```

Comment:
I'm concerned that this might be executed every time the migration flow is accessed and end up cleaning important data. Have you considered this? 
I wonder if this can introduce some other faulty behavior. Since we have to clean this it, what if we need the information later on in the flow?

Reply:
I would say it is safe to use. It is common practices on other flows to reset the store on the same we are doing here. Like on `importHostedSiteFlow` e `migrationSignup` flows. This is important data, but is populated during the steps. Reset it every time avoid other flows like `setup/onboarding` to contaminate the migration flow.